packages/react-components/src/hooks/useAlarms/hookHelpers/predicates.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  AlarmAssetModelRequest,
+  AlarmAssetRequest,
+  AlarmCompositeModelRequest,
+  AlarmInputPropertyRequest,
+  AlarmRequest,
+} from '../types';
+
+export const isAssetModelRequest = (
+  alarmRequest: AlarmRequest
+): alarmRequest is AlarmAssetModelRequest => Boolean(alarmRequest);
```

Comment:
This is basically casting, as you are simply saying a type is another type if it is truthy. Same with the other type guard.

Reply:
Yes I updated this afterward to replace `Boolean(request)` with a check for a field. In this case it is `alarmRequest.assetId === undefined` would confirm that it is an assetModel request.